assets/romfs/i18n/de/gui.json

```diff
@@ -0,0 +1,446 @@
+    "TOUGH_CONTEST_VALUE": "Stärke Wettbewerbs-Eigenschaft Wert",
+    "SHEEN_CONTEST_VALUE": "Glanz Wettbewerbs-Eigenschaft Wert",
+    "MOVE_1": "Attacke 1",
+    "MOVE_2": "Attacke 2",
+    "MOVE_3": "Attacke 3",
+    "MOVE_4": "Attacke 4",
+    "MOVE_1_PP": "Attacke 1 AP",
+    "MOVE_2_PP": "Attacke 2 AP",
+    "MOVE_3_PP": "Attacke 3 AP",
+    "MOVE_4_PP": "Attacke 4 AP",
+    "MOVE_4_PP_UPS": "Attacke 4 AP Ups",
+    "IVS_EGG_AND_NICKNAMED_FLAGS": "DVs, Ei, und Spitznamen Markierung",
+    "HOENN_RIBBONS": "Hoenn Bänder",
+    "GENDER_&_FORMS": "Geschlecht & Formen",
+    "MISC_FLAGS": "Verschiedene Markierungen",
+    "ORIGIN_GAME": "Herkunftsspiel",
+    "EGG_DATE": "Ei Datum",
+    "ENCOUNTER_TYPE": "Begegnugs Typ",
+    "CAPSULE_INDEX_(SEALS)": "Kapsel Index (Sticker)",
+    "MAIL_MESSAGE_+_OT_NAME": "Brief Nachricht + OT Name",
+    "SHINY_LEAVES_(HGSS)": "Glänzende Blätter (HGSS)",
+    "EGG_LOCATION_(PLATINUM)": "Ort des Ei-Erhalts (Platinum)",
+    "MET_LOCATION_(PLATINUM)": "Ort der Begegnung (Platinum)",
+    "EGG_LOCATION_(DIAMOND_&_PEARL)": "Ort des Ei-Erhalts (Diamond & Pearl)",
+    "MET_LOCATION_(DIAMOND_&_PEARL": "Ort der Begegnung (Diamond & Pearl",
+    "HGSS_POKEBALL": "HGSS Pok\u00e9 Ball",
+    "SEAL_COORDINATES": "Sticker Koordinaten",
+    "REPORT_THIS_TO_FLAGBREW": "Melde dies FlagBrew",
+    "FATEFUL_ENCOUNTER": "Schicksalhafte Begegnung",
+    "SECRET_SUPER_TRAINING": "Geheimtraining",
+    "EGG": "Ei",
+    "NICKNAMED": "Benannt",
+    "FEMALE_OT": "Weiblicher OT",
+    "POISONED": "Vergiftet",
+    "BURNED": "Verbrannt",
+    "FROZEN": "Eingefroren",
+    "PARALYZED": "Paralysiert",
+    "TOXIC": "Toxin",
+    "HIDDEN_ABILITY?": "Versteckte Fähigkeit?",
+    "NS_POKEMON?": "N's Pokemon?",
+    "HEX_SELECTED_BYTE": "Ausgewählter Byte:",
+    "WC_LGPE": "LGPE Speicherstände speichern keine Wunderkarten!",
+    "NOT_A_BUG": "Dies ist kein Bug.",
+    "WC_SWITCH": "\uE004 / \uE005 um zu mehreren WK zu wechseln.",
+    "LANGUAGES": "Sprachen",
+    "OVERWRITE_WC": "Überschreibe Wunderkarten",
+    "ADAPT_LANGUAGE": "Passe Sprache an WK an",
+    "INJECT_TO_SLOT": "Beim Slot einfügen",
+    "START_TO_INJECT": "Drück START zum Einfügen",
+    "HELD_ITEM": "Getragenes Item",
+    "GAME": "Spiel",
+    "DATE": "Datum",
+    "YOUR_OT_NAME": "Dein OT Name",
+    "NA": "N/A",
+    "BP": "GP",
+    "WC_CHANGE_SLOT": "Drück \uE000 um den Slot zu wechseln",
+    "WC_DUMP2": "Drück \uE002 um die Wunderkarte zu dumpen",
+    "WC_DUMP1": "Drück \uE000 um die Wunderkarte zu dumpen",
+    "WC_INST1": "Drück \uE000 um fortzufahren oder \uE001 um zurückzukehren.",
+    "EVENT_DATABASE": "Event Databank",
+    "FAILED_OPEN_DUMP": "Konnte die Datei zum dumpen nicht öffnen!",
+    "SPATK.": "Sp.-Ang.",
+    "SPDEF.": "Sp.-Vert.",
+    "NEUTRAL": "Neutral",
+    "STORAGE": "Lagerung",
+    "EDITOR": "Editor",
+    "EVENTS": "Events",
+    "SCRIPTS": "Skripte",
+    "BAG": "Beutel",
+    "SETTINGS": "Einstellungen",
```

Comment:
Shorten this line a little bit. i.e. 'Options' instead of 'Settings' if that is a shorter word